README.md

```diff
@@ -18,9 +18,14 @@ Running the Tests
 =================
 
 The tests are designed to be run from your local computer. The test
-environment requires Python 2.7+ (but not Python 3.x). You will also
-need a copy of OpenSSL. Users on Windows should read the
-[Windows Notes](#windows-notes) section below.
+environment requires [Python 2.7+](http://www.python.org/downloads) (but not Python 3.x).
+Leave the default install settings so Python will be installed to the c:\python2x directory.
+
+Be sure to add that directory to your `%Path%`
+[Environment Variable](http://www.computerhope.com/issues/ch000549.htm).
```

Comment:
The `%Path%` syntax is Windows-specific, right? And the http://www.computerhope.com/issues/ch000549.htm info is also Windows-specific. So it seems like this should be moved to that [Windows Notes](https://github.com/w3c/web-platform-tests/blob/master/README.md#windows-notes) section.